.vscode/settings.json

```diff
@@ -107,5 +109,9 @@
         "Xylopyranose",
         "Xylulose"
     ],
-    "rust-analyzer.showUnlinkedFileNotification": false
-}
\ No newline at end of file
+    "rust-analyzer.showUnlinkedFileNotification": false,
+    "rust-analyzer.server.extraEnv": {
+        "PYO3_PYTHON": "./.venv/Scripts/python.exe"
+    },
+    "rust-analyzer.linkedProjects": ["Cargo.toml", "rustyms-py/Cargo.toml"]
```

Comment:
Tiny question as it was added as a project in the root Cargo.toml, do we still need this setting set up?

Reply:
Not sure. I get this error without it:

```
ERROR rust_analyzer::main_loop: FetchWorkspaceError:
rust-analyzer failed to discover workspace
```

For now, I removed it from git again.

Reply:
If you get an error without then it is good to leave it in.